notifications.bs

```diff
@@ -718,7 +722,7 @@ constructor steps are:
 <h3 id=static-members>Static members</h3>
 
 <p>The static <dfn attribute for=Notification><code>permission</code></dfn> getter steps are to
-return the <a>permission</a> for the <a>entry settings object</a>'s
+return the <a>permission</a> for <a>this</a>'s <a>relevant settings object</a>'s
```

Comment:
This is static so it doesn't work. You want the `<a>current settings object</a>`.

Reply:
Hmm this probably is preexisting in requestPermission() somehow.

Reply:
I fixed both, thanks for spotting that!